telemetry/specs/INT.mdk

```diff
@@ -910,6 +913,190 @@ from (shim header length \* 4).
 For INT over Geneve it is 8 bytes subtracted from (length in Geneve tunnel
 option header \* 4).
 
+
+INT Metadata Header and Metadata Stack (Version = 2):
+`
+    0                   1                   2                   3
+    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |Ver = 2|Rep|C|E|M|S|    Reserved     | Hop ML  |RemainingHopCnt|
```

Comment:
With "Domain Specific" expanding into "Reserved" (i.e. another 16 bits), the S bit is no longer needed in this first part of the header.